app/assets/javascripts/navbar.js

```diff
@@ -28,15 +28,19 @@ setupPage = function() {
         })
     }
 
+    $('.navbar').removeClass("navbar-fixed-top");
     changeNavbarTransparency();
 }
 
 changeNavbarTransparency = function() {
-    if($('body').hasClass('background')){
-        $('.navbar-fixed-top').addClass('transparent');
-    } else {
-        $('.navbar-fixed-top').removeClass('transparent');
-    }
+     if($('body > div').hasClass('background')){
+         $('.navbar').addClass('transparent');
+         $('.navbar').removeClass('not-transparent');
+     }
+     else {
+         $('.navbar').removeClass('transparent');
+         $('.navbar').addClass('not-transparent');
```

Comment:
Is this else clause really necessary? I can't find any usage of the class `not-transparent` in the css and the navbar element doesn't have the `transparent` class assigned by default so both statements seem unnecessary.